internal/scan/node_scan.go

```diff
@@ -33,15 +33,15 @@ func RunNodeScan(ctx context.Context, cfg *types.Config) []*types.ScanResults {
 	root := cfg.NodeScan
 	rpms, _ := rpm.GetAllRPMs(ctx, root)
 	for _, pkg := range rpms {
-		tag := NewTag(pkg)
-		files, err := rpm.GetFilesFromRPM(ctx, root, pkg)
+		tag := NewTag(pkg.Name)
+		files, err := rpm.GetFilesFromRPM(ctx, root, pkg.NVRA)
```

Comment:
We use name (e.g. `glibc`) for config exceptions, and NVRA (e.g. `glibc-2.28-189.5.el8_6.x86_64`) for getting the list of files (because more than 1 glibc may be installed).

Reply:
Potentially, we can modify the code to be able to use both the name and the NVRA (name-version-release.architecture) in the exclusion lists; until there's a need for that, I'd like to not do it.